### PR TITLE
codeintel: add additional relationship data to snapshot view

### PR DESCRIPTION
**Pull request by Strum355**
Adds relationship data to scip snapshot UI in blob view

## Test plan

image

### CODE REVIEW

File: cmd/frontend/graphqlbackend/codeintel.codenav.graphql [hunk 1]
```diff
@@ -259,6 +259,8 @@ type SnapshotData {
     The formatted SCIP symbol string.
     """
     data: String!
+
+    additional: [String!]
```
Comment (efritz) [inline cmd/frontend/graphqlbackend/codeintel.codenav.graphql:263]: Can we get a docstring?

File: enterprise/internal/codeintel/codenav/service.go [hunk 2]
```diff
@@ -1443,14 +1446,58 @@ func (s *Service) SnapshotForDocument(ctx context.Context, repositoryID int, com
...
 		snap.WriteRune(' ')
 		snap.WriteString(formatted)
 
+		snapshotData.Symbol = snap.String()
+
+		// hasOverrideDocumentation := len(occ.OverrideDocumentation) > 0
+		// if hasOverrideDocumentation {
+		// 	documentation := occ.OverrideDocumentation[0]
+		// 	writeDocumentation(&b, documentation, prefix, true)
+		// }
```
Comment (efritz) [inline enterprise/internal/codeintel/codenav/service.go:1464]: Got a lotta comments in here 🦖

File: cmd/frontend/graphqlbackend/codeintel.codenav.graphql (hunk 1 above)
Comment (Strum355) [reply cmd/frontend/graphqlbackend/codeintel.codenav.graphql:263]: I find it completely useless, at least for scip-go its just `documentation ```go`

Comment (efritz) [reply cmd/frontend/graphqlbackend/codeintel.codenav.graphql:263]: What can additional contain though? just more data?

Comment (Strum355) [reply cmd/frontend/graphqlbackend/codeintel.codenav.graphql:263]: Ya, theyre just opaque strings  that are added in addition to the base definition/reference one

**Pull request merged**